--- a/APKBUILD	2024-05-15 18:32:45
+++ b/APKBUILD	2024-05-15 18:32:30
@@ -11,7 +11,7 @@
 url="https://axoflow.com/docs/axosyslog/"
 arch="all"
 license="GPL-2.0-or-later"
-options="!check" # unit tests require https://github.com/Snaipe/Criterion with deps
+options="!check !strip" # unit tests require https://github.com/Snaipe/Criterion with deps
 makedepends="
 	bison
 	bpftool
@@ -85,6 +85,7 @@
 build() {
 	CFLAGS="$CFLAGS -flto=auto" \
 	./configure \
+		--enable-debug \
 		--prefix=/usr \
 		--sysconfdir=/etc/syslog-ng \
 		--localstatedir=/var/lib/syslog-ng \
